packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -126,49 +145,6 @@ describe("LabeledField", () => {
         expect(error).toBeInTheDocument();
     });
 
-    it("LabeledField adds the correctly formatted id to label's htmlFor", () => {
```

Comment:
Removed since this is a duplicate of this test: https://github.com/Khan/wonder-blocks/blob/2b809b7d46afbf8bfab02559afb5778ad3723992/packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx#L436-L453